provisioner/ibm-s3fs-provisioner.go

```diff
@@ -67,6 +68,7 @@ type pvcAnnotations struct {
 	AccessPolicyAllowedIps  string `json:"ibm.io/access-policy-allowed-ips,omitempty"`
 	AddMountParam           string `json:"ibm.io/add-mount-param,omitempty"`
 	QuotaLimit              string `json:"ibm.io/quota-limit,omitempty"`
+	BucketVersioning        string `json:"ibm.io/bucket-versioning,omitempty"`
```

Comment:
It should be a boolean annotation.
Replace `BucketVersioning        string `json:"ibm.io/bucket-versioning,omitempty"`` --> `BucketVersioning        bool `json:"ibm.io/bucket-versioning,string,omitempty"``